logsearchapi/server/query.go

```diff
@@ -101,6 +102,15 @@ func searchQueryFromRequest(r *http.Request) (*SearchQuery, error) {
 		timeStart = &ts
 	}
 
+	var timeEnd *time.Time
+	if timeParam := values.Get("timeEnd"); timeParam != "" {
+		ts, err := parseSQTimeString(timeParam)
+		if err != nil {
+			return nil, fmt.Errorf("Invalid start date (must be RFC3339 format): %s", timeParam)
```

Comment:
should we log the `err ` ?

Reply:
we don't log the errors in place on this code, but I agree it'd be nicer if we do it all over log search api